### DISCUSSION TITLE
Can we add different lightings to the model?

### DISCUSSION THREAD

**Discussion by codepushed** [google/model-viewer#4181] [Q&A]
Like three js have options for adding a couple of lighting to the model such as Rect Area light, spotlight, ambient light, and others. Can we do the same with the modelviewer?

**Comment by elalish** [comment]
We've decided against this because it would expand our API surface too much. Also, they get expensive to render pretty quickly. The effects of all of these lights (and more) can be reproduced quite well with a single environment map, so long as it has very high dynamic range. Give it a try and let us know if there's anything that's hard to reproduce.

**Reply by micsun-al** [reply]
I have a football helmet and the inside is black. I made a custom environment map that shines a light from the bottom, but most of the features inside the helmet are a bit hard to see without a light inside the helmet.

![image](https://github.com/user-attachments/assets/504fe399-e872-41b9-a066-3c4eac3c6ae6)

It'd be really nice if we could have a point light...please? :)
